### PR TITLE
liblaxjson 1.0.5 (new formula)

### PR DESCRIPTION
**Pull request by CalebMorris**
Create new formula for andrewrk/liblaxjson v1.0.5

### CODE REVIEW

File: Library/Formula/liblaxjson.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (bfontaine) [inline Library/Formula/liblaxjson.rb]: Please remove this empty line.

File: Library/Formula/liblaxjson.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class Liblaxjson < Formula
+  desc "C library for parsing relaxed JSON config file"
+  homepage "https://github.com/andrewrk/liblaxjson"
+  url "https://github.com/andrewrk/liblaxjson/archive/1.0.5.tar.gz"
+  version "1.0.5"
```
Comment (bfontaine) [inline Library/Formula/liblaxjson.rb]: You can remove this line; it’s detected from the URL.

File: Library/Formula/liblaxjson.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "ffc495b5837e703b13af3f5a5790365dc3a6794f12f0fa93fb8593b162b0b762"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "mkdir", "build"
+    cd "build" do
```
Comment (bfontaine) [inline Library/Formula/liblaxjson.rb]: Use `mkdir "build" do`

File: Library/Formula/liblaxjson.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "mkdir", "build"
+    cd "build" do
+      system "echo", "`pwd`"
+      system "cmake", *testpath, *std_cmake_args
```
Comment (bfontaine) [inline Library/Formula/liblaxjson.rb]: You can’t rely on `cmake` being available outside of the `install` method. See [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/599bc69a0e2e20a66d0234b56ac14a65f2e9f7a0/Library/Formula/tinyxml2.rb#L23-L33) for an example of a library formula with a good test.

**Comment by CalebMorris** [comment]
Audio shows response:
"not notable enough".
Since it's only used my 2 other people as far as I'm aware I'll gonna just keep it local unless it becomes more popular.

Thank you for the assistance in learning some more about the workflow of a new formula though.

**Pull request closed without merging**